webapp/tbapi/tbapi/models/tbapp.py

```diff
@@ -0,0 +1,66 @@
+from sqlalchemy import Column
+from sqlalchemy import (
+    Integer, String, Text
+)
+from sqlalchemy.orm import Session
+
+import json
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from .base import db, SerializeMixin
+
+class Note(db.Model, SerializeMixin):
+    __tablename__ = 'tbapi_notes'
+
+    id = Column(Integer, primary_key=True)
+    patient_id = Column(Integer, nullable=False)
+    text = Column(String(1000), nullable=False)
+    author_id = Column(Integer, nullable=False)
+    created = Column(String(100), nullable=False)
+    lastmod = Column(String(100), nullable=False)
+    
+    @classmethod
+    def get_by_patient_id(cls, patient_id):
+        notes = cls.query.filter_by(patient_id=patient_id).all()
+        return notes
+
+    @classmethod
+    def get_by_note(cls, id):
+        notes = cls.query.filter_by(id=id).all()
+        return notes
+
+    @classmethod
+    def post(cls, noteObj):
+        
+        notes = cls.query.filter_by(
+            patient_id=noteObj['patient_id'], 
+            text=noteObj['text'], 
+            created=noteObj['created'], 
+            author_id=noteObj['author_id'], 
+            lastmod=noteObj['lastmod']).all()
```

Comment:
It seems like we'd want to remove
the `created` and `lastmod` parameters from this query.

Those timestamps will change with every request,
so including the timestamps in this query
means we'll never find the original notes.

Demo:
```bash
> # One timestamp
> curl \
    -i \
    -X POST \
    http://localhost:5061/api/v1.0/notes \
    -H 'Content-Type: application/json' \
    -d '{
    "author_id": 0,
    "created": "2018-09-20 17:49:13",
    "lastmod": "2018-09-20 17:49:13",
    "patient_id": 88,
    "text": "text note 001"
    }'
HTTP/1.0 200 OK
Content-Type: application/json
Content-Length: 2
Access-Control-Allow-Origin: *
Server: Werkzeug/0.14.1 Python/3.6.6
Date: Mon, 08 Oct 2018 18:21:53 GMT

6
> # Slightly different timestamp
> curl \
    -i \
    -X POST \
    http://localhost:5061/api/v1.0/notes \
    -H 'Content-Type: application/json' \
    -d '{
    "author_id": 0,
    "created": "2018-09-20 17:49:14",
    "lastmod": "2018-09-20 17:49:14",
    "patient_id": 88,
    "text": "text note 001"
    }'
HTTP/1.0 200 OK
Content-Type: application/json
Content-Length: 2
Access-Control-Allow-Origin: *
Server: Werkzeug/0.14.1 Python/3.6.6
Date: Mon, 08 Oct 2018 18:21:53 GMT

7
```